web/src/main/java/org/akaza/openclinica/control/admin/CreateCRFVersionServlet.java

```diff
@@ -686,7 +686,41 @@ public ResponseOptionBean hasDifferentOption(ResponseSetBean oldRes, ResponseSet
         ArrayList newOptions = newRes.getOptions();
         if (oldOptions.size() != newOptions.size()) {
             // if the sizes are different, means the options don't match
-            return null;
+        	//OC-15058, but still need to compare the "overlapped" part
+        	int smallerSize =0;
+        	if(oldOptions.size() < newOptions.size()) {
+        		smallerSize = oldOptions.size();
+        	}else {
+        		smallerSize = newOptions.size();
+        	}        	
```

Comment:
If the sizes are different, should we throw an error?

Reply:
What happens if the order of the values change?

Reply:
Get  more feedback about the requirements from CS team:
If the sizes are different, should we throw an error? ---- NO, still allow to process.
What happens if the order of the values change? --- Order doesn't matter as long as the coded values are kept the same
Did more code changes after reconsider above feedback.